AEPCore/Sources/rules/LaunchRulesEngine.swift

```diff
@@ -128,12 +132,19 @@ public class LaunchRulesEngine {
                     eventData = modifiedEventData
 
                 case LaunchRulesEngine.CONSEQUENCE_TYPE_DISPATCH:
+                    if let unwrappedDispatchCount = dispatchChainCount, unwrappedDispatchCount > LaunchRulesEngine.MAX_CHAINED_CONSEQUENCE_COUNT {
```

Comment:
Is the intention to let this trigger the dispatch consequence twice or should the condition be unwrappedDispatchCount >= LaunchRulesEngine.MAX_CHAINED_CONSEQUENCE_COUNT?

Reply:
We can do it either way. What is more readable to you? 
The max count is really arbitrary, so if you prefer to use `>=` and set the max count to 2, that's fine.
Also, is 2 iterations a good max count or is it too conservative?  There could be valid use cases to tigger a rule off of a dispatched event. If we are using this check as a failsafe to prevent an infinite loop, then we should set the count higher. If we want to prevent any chaining of events dispatched from rules, then we can set the count to 1 or 2.

Reply:
I find it more intuitive with `>= MAX_COUNT`. I think we can start with 1 and relax this later if needed.